localstripe/resources.py

```diff
@@ -1625,6 +1614,26 @@ def last_payment_error(self):
                     'message': charge.failure_message,
                 }
 
+    @classmethod
+    def _api_create(cls, confirm=None, off_session=None, **data):
+        confirm = try_convert_to_bool(confirm)
+        off_session = try_convert_to_bool(off_session)
+        try:
+            if confirm is not None:
+                assert type(confirm) is bool
+            if off_session is not None:
+                assert type(off_session) is bool
+                assert confirm is True
+        except AssertionError:
+            raise UserError(400, 'Bad request')
+
+        obj = super()._api_create(**data)
+
+        if confirm:
+            cls._api_confirm(obj)
```

Comment:
We should `_api_confirm` with ID instead of entire object.

```python
if confirm:
    cls._api_confirm(obj.id)
```

Reply:
Ah, of course! Thanks.

Updated:
```diff
--- a/localstripe/resources.py
+++ b/localstripe/resources.py
@@ -1630,7 +1630,7 @@ class PaymentIntent(StripeObject):
         obj = super()._api_create(**data)
 
         if confirm:
-            cls._api_confirm(obj)
+            cls._api_confirm(obj.id)
 
         return obj
 
```